File: tools/template-ts/src/task.ts
/**
 * Task implementation — REPLACE THIS FILE.
 *
 * Must export:
 *   SERVERS: string[]  — MCP server names to connect
 *   runTask(clients, config): Promise<void>  — the task logic
 */

import type { Clients } from "./tools.js";

export const SERVERS: string[] = [];

export async function runTask(
  _clients: Clients,
  _config: Record<string, unknown>,
): Promise<void> {
  console.log("No task implemented. Edit src/task.ts.");
}

File: tools/template-py/task.py
"""Task definition — REPLACE THIS FILE with your own logic."""

from typing import Any

# Which MCP servers to connect (filtered from config automatically).
# Available: google, linkedin, web, filesystem, github, anthropic-admin, interview-assist
SERVERS = ["google", "linkedin", "web", "filesystem", "github", "anthropic-admin", "interview-assist"]


async def run_task(clients: dict[str, Any], config: dict) -> None:
    """Placeholder — replace with your collect → score → report logic."""
    print("\n[Task placeholder]")
    print("Replace task.py with your own logic.")
    print(f"Connected servers: {list(clients.keys())}")
